libs/remix-tests/sol/tests.sol.ts

```diff
@@ -8,14 +8,21 @@ library Assert {
     string message
   );
 
+  event AssertionEventUint(
+    bool passed,
+    string message,
+    uint256 returned,
+    uint256 expected
+  );
+
   function ok(bool a, string memory message) public returns (bool result) {
     result = a;
     emit AssertionEvent(result, message);
   }
 
-  function equal(uint a, uint b, string memory message) public returns (bool result) {
```

Comment:
why did you change the type to `uint256`?

Reply:
`uint` is alias of `uint256`. `uint256` is required in event details here: https://github.com/ethereum/remix-project/blob/expectVsReturn/libs/remix-tests/src/assertionEvents.ts#L8 while decoding the event data, so I put it everywhere